docker-compose.yaml

```
@@ -0,0 +1,107 @@
+version: '3.12'
+services:
+
+  materials:
+    container_name: materials
+    build:
+      context: .
+      dockerfile: ./Dockerfile
+    command: ./docker_commands.sh
+    volumes:
+      - .:/code
+    ports:
+      - "8000:8000"
+    env_file:
+      - ./.env
+    depends_on:
+      db_mypedia:
+        condition: service_healthy
+      celery:
+        condition: service_started
+      celery_beat:
+        condition: service_started
+    networks:
+      - local
+      - global
+
+  db_materials:
+    container_name: db_materials
+    image: postgres:16.4
+    env_file:
+      - ./.env
+    ports:
+      - "5432:5432"
+    environment:
+      - POSTGRES_DB=${DB_NAME}
+      - POSTGRES_USER=${DB_USER}
+      - POSTGRES_PASSWORD=${DB_PASSWORD}
+    volumes:
+      - ./data/postgres:/var/lib/postgresql/data
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U ${DB_USER}"]
```

Тут правильнее еще указывать название базы
```suggestion
      test: ["CMD-SHELL", "pg_isready -U ${DB_USER} -d ${DB_NAME}"]
```